src/components/Comment/AddComment/StarInput/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { useState, useEffect } from "react";
+import fillStarIcon from "../../../../assets/svg/rateFillStar.svg";
+import emptyStarIcon from "../../../../assets/svg/emptyStar.svg";
+import halfStarIcon from "../../../../assets/svg/halfStar.svg";
+
+interface Props {
+  star: number;
+  setStar: (star: number) => void;
```

Comment:
```suggestion
  setStar: Dispatch<SetStateAction<number>>;
```
setState 함수 타입은 이렇게 선언하는 게 정확해 (React)